e2e-tests/node-client.js

```diff
@@ -94,9 +106,9 @@ class NodeClient {
      */
     async unsafeTransfer(to, amount, nonce, privateKey){
         const from = await Utils.getKeyringPair(privateKey);
-        const U64 = UInt.with(64, "U64");
-        amount = u8aToHex((new U64(TypeRegistry, amount)).toU8a());
-        const signedTx = await Utils.signTransaction(from, to, amount, nonce);
+        const rawNonce = !nonce ? "0x" : u8aToHex((new U64(TypeRegistry, nonce)).toU8a());
```

Comment:
Normally we are checking for the straight case and after that the negative. Wouldn't it be better if we refactor it to:
```
const rawNonce = nonce ? u8aToHex((new U64(TypeRegistry, nonce)).toU8a()) : "0x"
```

The same thing can be applied to the `amount`